### PR TITLE
Add hint metrics

### PR DESCRIPTION
**Pull request by lyubent**
Adds hint metrics for:
- total hints in progress - hint messages attempting to be handed off (since the Cassandra process' startup)
- total hints - total hint messages created (since the Cassandra process' startup )

These metrics are being scraped from`org.apache.cassandra.metrics:type=Storage`.

### CODE REVIEW

File: jolokia/client.go [hunk 1]
```diff
@@ -442,6 +442,28 @@ func (c *jolokiaClient) StorageStats() (StorageStats, error) {
...
+	if err != nil {
+		return HintStats{}, fmt.Errorf("err reading CQL stats: %v", err)
+	}
+
+	stats := HintStats{}
+	v.Get("value").GetObject().Visit(func(key []byte, val *fastjson.Value) {
+		attributes := extractAttributes(string(key))
+		switch attributes["name"] {
+		case "TotalHintsInProgress":
+			// PreparedStatementsEvicted
```
Comment (suhailpatel) [inline jolokia/client.go]: Can this comment be removed please as it seems irrelevant?

File: README.md [hunk 2]
```diff
@@ -140,6 +140,15 @@ These metrics come from Cassandra's storage service which keeps track of the clu
...
 | `seastat_storage_node_status` | Status (`live`, `unreachable`, `joining`, `moving`, `leaving`) of each node in the cluster (tagged by node and status) | Gauge |
 
+## Hint Metrics
+
+These metrics come from the [Storage](https://cassandra.apache.org/doc/latest/operating/metrics.html#storage-metrics) metric which keeps track of hints, node load and storage exceptions.
+
+| Name          | Description   | Type |
+| ------------- | ------------- | ---- |
+|TotalHintsInProgress|Number of hints attemping to be sent currently.|Counter|
+|TotalHints|Number of hint messages written to this node since [re]start. Includes one entry for each host to be hinted per hint.|Counter|
```
Comment (suhailpatel) [inline README.md]:
```suggestion
| `seastat_total_hints` | Number of hint messages written to this node since [re]start. Includes one entry for each host to be hinted per hint. | Counter |
| `seastat_total_hints_in_progress` | Number of hints attempting to be sent currently from this node. | Gauge |
```

File: jolokia/client.go [hunk 3]
```diff
@@ -442,6 +442,28 @@ func (c *jolokiaClient) StorageStats() (StorageStats, error) {
...
+			stats.TotalHintsInProgress = Counter(val.Get("Count").GetInt64())
```
Comment (suhailpatel) [inline jolokia/client.go]: Hmm are we sure this is a Counter. I did see in the docs it says a Counter but if it decreases then we should export it as a gauge. Since it's an InProgress number, it feels like it could go down too?

**Review by suhailpatel — COMMENTED** [review]
Thanks for the addition, a few suggested tweaks and a check on whether the `seastat_total_hints_in_progress` should be a gauge

File: jolokia/client.go (hunk 3 above)
Comment (lyubent) [reply jolokia/client.go]: Yes, the metric can go up then down after the hint finishes replaying. It's a simple enough change. I'll push the update now.

**Review by suhailpatel — APPROVED** [review]
Sweet, i'll merge and cut a new release. Thanks!
